packages/container/tests/container_test.js

```diff
@@ -535,3 +517,27 @@ QUnit.test('Lazy injection validations are cached', function() {
   container.lookup('apple:main');
   container.lookup('apple:main');
 });
+
+if (isEnabled('ember-container-inject-owner')) {
+  QUnit.test('A deprecated `container` property is appended to every instantiated object', function() {
+    const registry = new Registry();
+    const container = registry.container();
+    const PostController = factory();
+    registry.register('controller:post', PostController);
+    const postController = container.lookup('controller:post');
+
+    expectDeprecation(function() {
+      Ember.get(postController, 'container');
```

Comment:
we should likely also test `postController.container` as typically the container can be accessed as a raw property.